File: Tree/basicTree.java
package Tree;
import java.util.ArrayList;

class TreeNode{
    String data;
    ArrayList<TreeNode> children;
    public TreeNode(String data)
    {   
        this.data=data;
        this.children= new ArrayList<TreeNode>();
    }
    void addChild(TreeNode node)
    {
        this.children.add(node);
    }
    String print(int level)
    {
        String str=" ".repeat(level)+ this.data+"\n";
        for(TreeNode node : this.children){
            str += node.print(level+1);
        };
        return str;
    }
}

public class basicTree {
    public static void main(String[] args)
    {
        TreeNode drinks = new TreeNode("Drinks");
        TreeNode hot = new TreeNode("Hot");
        TreeNode cold = new TreeNode("Cold");
        TreeNode tea = new TreeNode("Tea");
        TreeNode coffee = new TreeNode("Coffee");
        TreeNode wine = new TreeNode("Wine");
        
        drinks.addChild(hot);
        drinks.addChild(cold);
        hot.addChild(tea);
        hot.addChild(coffee);
        cold.addChild(wine);

        System.out.println(drinks.print(0));
    }
}

File: Tree_Python/basic_tree.py
class treenode:
    def __init__(self,data,children=[]):
        self.data=data
        self.children=children
    
    def __str__(self,level=0):
        res=(" " * level)+str(self.data)+"\n"
        for child in self.children:
            res=res+child.__str__(level+1)
        return res

    def addchildren(self,treenode):
        self.children.append(treenode)
    
tree = treenode("drinks",[])
cold = treenode("cold",[])
hot = treenode("hot",[])

tree.addchildren(cold)
tree.addchildren(hot)
print(tree)